src/Network/Haskoin/Store/Data.hs

```diff
@@ -768,6 +865,14 @@ instance BinSerial PeerInformation where
         put $ show a
```

Comment:
Yo uhave here the `peerServices :: Word64` record and the `peerAddress :: SockAddr` records. Combine these two into a `NetworkAddress` (see [here](https://github.com/haskoin/haskoin-core/blob/dda2da7050d088c0761cea833fdfbbe85fa02483/src/Network/Haskoin/Network/Common.hs#L192-L197)) and use the existing `Serialize` instance.